python/setup.py

```diff
@@ -131,7 +131,11 @@ def has_ext_modules(self):
         "redis",
         # The six module is required by pyarrow.
         "six >= 1.0.0",
-        "flatbuffers"
+        "flatbuffers",
+        "feather-format",
+        "lxml",
+        "openpyxl",
+        "xlrd"
```

Comment:
These should probably not be added to the `setup.py`. We can have add them as a separate entry under `extras` above.

The requirements here should be fairly minimal, e.g., in theory we would like to require `jupyter` here since we need it for the UI, but running `pip install jupyter` doesn't work universally enough to be a dependency here.